src/test/java/uk/gov/register/functional/LoadSerializedFunctionalTest.java

```diff
@@ -26,17 +25,26 @@
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsEqual.equalTo;
 import static org.junit.Assert.assertThat;
-import static uk.gov.register.functional.db.TestDBSupport.*;
 
 public class LoadSerializedFunctionalTest {
-    public static final int APPLICATION_PORT = 9000;
     private static final String registerName = "register";
 
     @Rule
     public TestRule wipe = new WipeDatabaseRule();
 
     @ClassRule
-    public static final RegisterRule register = new RegisterRule(registerName);
+    public static final RegisterRule register = new RegisterRule();
```

Comment:
can they not just be extra methods on the RegisterRule, or a separate helper class? (I prefer composition over inheritance :P )